couchexport/util.py

```diff
@@ -45,6 +45,10 @@ def clear_attachments(schema_or_doc):
         del schema_or_doc['_attachments']
     if schema_or_doc and 'case_attachments' in schema_or_doc:
         del schema_or_doc['case_attachments']
+    if schema_or_doc:
```

Comment:
Makes sense, after taking a look I don't see anything notably less sketchy to check for. I'll update.